pyre/bench/synth/getframe_inlined_callee_lasti_escape.py

```
@@ -0,0 +1,35 @@
+# An inlined callee returns its own escaped frame.  PyPy's dispatch loop writes
+# `last_instr` for both the `_getframe` call and the later return opcode.  The
+# JitCode walker bypasses that loop, so both transitions must be emitted on the
+# callee's own red frame.  Without them compiled iterations expose the frame
+# constructor's `last_instr = -1` sentinel (`f_lasti == -2`) alongside the
+# interpreted return coordinate.
+import sys
+
+N = 20000
+
+
+def leaf(x):
```

Ruff (0.16.1)

[warning] Unused function argument: `x`

(ARG001)